### ISSUE TITLE
Manual cropping

### ISSUE THREAD

**Issue by yephny** [Breta01/docus#33]
Cropping is arguably the most essential feature of a scanner app, other than scanning documents. The app is doing the cropping on its own, which doesn't really come out good often. Being able to crop the images manually, though still with image recognition (so that the whole side of the rectangle would be moving to the line the paper makes with the background when I move a corner of the cropping rectangle) would be super useful.

**Comment by Breta01** [comment]
Yes, that's definitely a great idea! Other two #34 and #35  as well. I think this should be the highest priority from the 3.

I appreciate any help with it. It can take some time before I will have time to work on it.
Thanks

**Comment by yephny** [comment]
Thanks for the app. If you haven't already, you can check out [OpenScan](https://github.com/Ethereal-Developers-Inc/OpenScan) as it can crop. Hope this helps, wish I had the skills to contribute.

**Comment by 5lttrz** [comment]
@Breta01 It would be great to have an option to fit page to image after cropping (if this gets implemented)

Currently none of the document scanner apps I have tried have option that resize the page to fit image, rather they come with white borders when cropped.

Docus looks promising, I hope you add an option for this suggestion.

**Comment by Breta01** [comment]
I totally agree that this would improve the app greatly. At the moment I am working on my other project, so it might take some time till I get to this 😓

**Comment by daraul** [comment]
It would also help when autocropping fails. I just tried to scan a payslip (not a regular A4 page) and it just did nothing. I then tried an A4 page and it worked perfectly.
